libs/wallet/src/use-eager-connect.ts

```diff
@@ -1,11 +1,14 @@
+import type { SnapConnector } from './';
 import { useVegaWallet } from './';
 import { useEffect, useState } from 'react';
 import type { VegaConnector } from './connectors/vega-connector';
 import { getConfig } from './storage';
+import { useEnvironment } from '@vegaprotocol/environment';
 
 export function useEagerConnect(Connectors: {
   [connector: string]: VegaConnector;
 }) {
+  const { VEGA_URL } = useEnvironment();
```

Comment:
chore (blocking): lets pass in origin as an argument for the hook. We'll need to publish this (and the connect dialog) soon, so we don't want to tie in deps that don't need to be

Reply:
Was just taking a look at this and its a bigger job than I expected and not worth cluttering this PR. I'm working on  https://github.com/vegaprotocol/frontend-monorepo/issues/4603 which will address this